core/src/main/java/com/javadiscord/jdi/core/Discord.java

```diff
@@ -152,6 +156,27 @@ public void start() {
         EXECUTOR.execute(discordRequestDispatcher);
     }
 
+    public void stop() {
+        if (this.webSocketManager != null) {
+            this.webSocketManager.stop();
+        }
+
+        ThreadPoolExecutor threadPool = (ThreadPoolExecutor) EXECUTOR;
```

Comment:
instead just change the type of `EXECUTOR`

Reply:
Preferably to `ExecutorService`.

Reply:
[e9264f3](https://github.com/javadiscord/java-discord-api/pull/100/commits/e9264f36468ab2126b3776b575092cd5a0ba2d4c)